app/helpers/github_webhook_helper.rb

```diff
@@ -0,0 +1,39 @@
+module GithubWebhookHelper
+
+  def handle_github_issue_opened(event)
+    return if !@author.nil?
+
+    author_payload = {
+      :author => {
+        :name => event["issue"]["title"],
+        :biography => event["issue"]["body"],
+        :github_issue_id => event["issue"]["id"]
+      }
+    }
+
+    params.merge!(author_payload)
+    self.create
+    book = Book.create(title: "Biography of #{event["issue"]["title"]}", author: @author, publisher: @author , price: 0.00)
+  end
+
+  def handle_github_issue_edited(event)
+    author_payload = {
+      :author => {
+        :name => event["issue"]["title"],
+        :biography => event["issue"]["body"]
+      }
+    }
+    
+    params.merge!(author_payload)
+    self.update
+    bio_book = @author.books.find_by(:price => 0)
```

Comment:
I think you would be better off just finding the first book here, this code is brittle and could lead to data corruption as soon as an author has more than one free book. Even finding the first book isn't _incredibly_ robust, but I think it's a better match for the project specs than just finding the first free book.